eng/scripts/Get-PackageVersions.ps1

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env pwsh
```

Comment:
Did you also `chmod +x` this file? We should get into that habit. I don't run `pwsh` regularly except when it's really useful to do so because it's slower to start.

Reply:
will `chmod +x` persist across windows changes?